var/spack/repos/builtin/packages/salmon/package.py

```diff
@@ -76,6 +76,10 @@ def patch(self):
                         'scripts/fetchPufferfish.sh')
             symlink('./salmon-v{0}.zip'.format(self.version),
                     './external/pufferfish.zip')
+            filter_file('INTERPROCEDURAL_OPTIMIZATION True', 
```

Comment:
Please add a comment explaining the patch and, if appropriate, provide a link to the issue.